scripts/pre-commit

```diff
@@ -1,35 +1,21 @@
 #!/bin/bash
 # WordPress pre-commit hook
+#
+# We assume that this will be called by a Composer or npm script so the
+# current working directory is the root of the project.
```

Comment:
Should we assume this yet? Do we need to break submodule installation if we don't have ti?

Reply:
This is mostly about being able to determine or guess `$DEV_LIB_PATH`. Previously we looked for two things:

- If the currently running `pre-commit` script is a symlink, we follow the link and set the `$DEV_LIB_PATH` to the directory of that original file path.

- If the current running `pre-commit` script is an actual system file, we set `$DEV_LIB_PATH` to the directory of that file path.

- If `$DEV_LIB_PATH` is not defined, assume that it is equal to `dev-lib`.

So you're right -- no need to remove the first two checks!